pkg/kubectl/sorting_printer.go

```diff
@@ -60,8 +61,28 @@ func (s *SortingPrinter) sortObj(obj runtime.Object) error {
 	if len(objs) == 0 {
 		return nil
 	}
+
+	if err := SortObjects(objs, s.SortField); err != nil {
+		return err
+	}
+
+	switch list := obj.(type) {
+	case *v1.List:
+		for ix := range objs {
+			data, err := v1.Codec.Encode(objs[ix])
```

Comment:
Yeah, as mentioned elsewhere (https://github.com/kubernetes/kubernetes/issues/15266) the trouble here is that `v1.RawExtension` is not a `runtime.Object` so in the trouble is that after parsing and sorting, I can't shove it back into the raw object without encoding.

I'm open to other solutions, I just don't see a great one.